packages/explorer/src/app/(home)/abi.ts

```diff
@@ -0,0 +1,63 @@
+export const abi = [
```

Comment:
ahh is this how you worked around the strongly typed json imports issue?

Reply:
changed to IBaseWorldABI from world package